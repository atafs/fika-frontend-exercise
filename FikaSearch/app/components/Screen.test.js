import React from 'react';
import renderer from 'react-test-renderer';
import Screen from './Screen';

describe(Screen.name, () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Screen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
