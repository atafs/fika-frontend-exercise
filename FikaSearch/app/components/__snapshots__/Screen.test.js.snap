// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screen renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "flex": 1,
        },
        undefined,
      ]
    }
  />
</RCTSafeAreaView>
`;
